catalog/app/containers/Bucket/Queries/Athena/CreatePackage.tsx

```diff
@@ -122,26 +122,21 @@ const useStyles = M.makeStyles((t) => ({
 }))
 
 interface CreatePackageProps {
-  bucket: string
   queryResults: requests.athena.QueryResultsResponse
 }
 
-export default function CreatePackage({ bucket, queryResults }: CreatePackageProps) {
+export default function CreatePackage({ queryResults }: CreatePackageProps) {
   const classes = useStyles()
   const [entries, setEntries] = React.useState<ParsedRows>({ valid: {}, invalid: [] })
   const addToPackage = AddToPackage.use()
-  const createDialog = usePackageCreationDialog({
-    bucket,
-    delayHashing: true,
-    disableStateDisplay: true,
-  })
+  const createPackage = useCreatePackage()
   const handleConfirm = React.useCallback(
     (ok: boolean) => {
       if (!ok) return
       addToPackage?.merge(entries.valid)
-      createDialog.open()
+      createPackage?.open()
```

Comment:
this would just silently fail if `createPackage` returns null/undefined. since this can only happen when there's a programmer error (uninitialized context), i think we can add assert/invariant after `const createPackage = useCreatePackage()` to ensure the context is initialized, e.g. `invariant(createPackage, "Package Dialog context wasn't initialized")`

Reply:
(this invariant can be added directly to the `use` function so that its return value is not nullable)